infrastructure/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "random_string" "password" {
+  length = 32
+  special = true
+}
+
+module "api" {
+  source   = "git@github.com/hmcts/terraform-module-webapp.git"
```

Comment:
@njrich28 should we specify version (tag) here, or are we ok to use latest master?

Reply:
use master